charts/clickhouse-serverless/tests/e2e.sh

```
@@ -0,0 +1,211 @@
+
+  helm_install \
+    -f "$CHART_DIR/tests/values-single.yaml" \
+    --set auth.existingSecret=ch-creds \
+    --set auth.secretKeys.passwordKey=password \
+    --set auth.password=""
+
+  pass "helm install with existing secret"
+
+  # Chart should NOT have created its own secret
+  kc get secret "${RELEASE}-clickhouse" &>/dev/null \
+    && fail "Chart-managed secret should not exist when existingSecret is set" \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)